service/service.py

```diff
@@ -22,6 +22,8 @@
 POST /wishlists - creates a new wishlist record in the database
 PUT /wishlists/{id} - updates a wishlist record in the database
 DELETE /wishlists/{id} - deletes a wishlist record in the database
+PUT /wishlists/{id}/enabled - enables a wishlist record in the database
```

Comment:
I think we should say ACTION here instead of put, as this is an Action API

Reply:
I think this change may not be necessary. If we say ACTION here, then it would not be consistent with the explanations of the paths of other APIs (line 20-24). For example, the CREATE API should not be POST /wishlists but CREATE /wishlists, but in the corresponding route method, we implement the HTTP method POST, not CREATE.

Reply:
Yes, I don't think it matters that much. I just saw that convention from other teams tbh.